src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/Gauges.hpp

```diff
@@ -3,7 +3,32 @@
 
 #pragma once
 
+#include <memory>
+#include <pup.h>
+
+#include "Parallel/CharmPupable.hpp"
+
 namespace GeneralizedHarmonic {
 /// \brief Gauge conditions for generalized harmonic evolution systems.
-namespace gauges {}
+namespace gauges {
+/// \brief Base class for GH gauge conditions.
+///
+/// Derived class must have:
```

Comment:
Incomplete documentation?

Reply:
I've rebased and squashed, updated the docs. However, the docs is written for the final state on my branch, so there are some functions and type aliases discussed that won't yet be merged. This just makes managing the PRs and commits easier on my end, and nobody can use this until all features are in anyway :)